examples/tf/rl2_ppo_halfcheetah.py

```diff
@@ -0,0 +1,80 @@
+"""Example script to run RL2 in HalfCheetah."""
+from garage import wrap_experiment
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
+from garage.experiment import task_sampler
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearFeatureBaseline
+from garage.sampler import LocalSampler
+from garage.tf.algos import RL2
+from garage.tf.algos.rl2 import RL2Env
+from garage.tf.algos.rl2 import RL2Worker
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianGRUPolicy
+
+
+@wrap_experiment
+def rl2_ppo_halfcheetah(ctxt=None, seed=1):
+    """Train PPO with HalfCheetah environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
+    """
+    set_seed(seed)
+    with LocalTFRunner(snapshot_config=ctxt) as runner:
+        max_path_length = 100
+        meta_batch_size = 10
+        n_epochs = 50
+        episode_per_task = 4
+
+        tasks = task_sampler.SetTaskSampler(lambda: RL2Env(
+            env=HalfCheetahVelEnv()))
+
+        env_spec = tasks.sample(1)[0]().spec
+        policy = GaussianGRUPolicy(name='policy',
+                                   hidden_dim=64,
+                                   env_spec=env_spec,
+                                   state_include_action=False)
+
+        baseline = LinearFeatureBaseline(env_spec=env_spec)
+
+        inner_algo_args = dict(
```

Comment:
can you create RL2PPO and RL2TRPO, which are basically just wrappers which have these args as constructor params rather than passing them as a dict? this will make a better API for uses (its also consistent with MAML).